ush/python/pygfs/task/land_analysis.py

```diff
@@ -35,6 +35,7 @@ def __init__(self, config):
         _res = int(self.config['CASE'][1:])
         _window_begin = add_to_datetime(self.runtime_config.current_cycle, -to_timedelta(f"{self.config['assim_freq']}H") / 2)
         _letkfoi_yaml = os.path.join(self.runtime_config.DATA, f"{self.runtime_config.RUN}.t{self.runtime_config['cyc']:02d}z.letkfoi.yaml")
+        _bufr2ioda_yaml = os.path.join(self.runtime_config.DATA, f"bufr_adpsfc_snow.yaml")
```

Comment:
```suggestion
```

I need to think of this a bit. I don't think this belongs in the constructor.

Reply:
Removed as suggested.